classes/class-wc-connect-shipping-label.php

```diff
@@ -362,6 +364,14 @@ protected function get_states_map() {
 		}
 
 		public function should_show_meta_box() {
+			if ( null === $this->show_metabox ) {
```

Comment:
@nabsul yep, seeing it twice using these steps:

1. Load order page at `wp-admin/post.php?post=123&action=edit`
2. See that both functions are called
3. Re-load the same page
4. See that both functions are called.

From what I can see, this happens because a new instance of `WC_Connect_Shipping_Label` is created each time.